geomesa-accumulo/geomesa-accumulo-datastore/src/test/scala/org/locationtech/geomesa/accumulo/index/FilterHelperTest.scala

```diff
@@ -353,4 +356,15 @@ class FilterHelperTest extends Specification with Mockito with Logging {
       result.asInstanceOf[KnownCost].cost mustEqual 5
     }
   }
+
+  "tryMergeGeoms" should {
+    "intersect BBOX geoms" >> {
```

Comment:
Looks like IDL is handled upstream of this case match.  I am unable to produce an IDL query (viewport spanning the IDL or poly spanning the IDL) that doesn't get handled upstream.